apps/challenges/aws_utils.py

```diff
@@ -669,6 +672,9 @@ def scale_resources(challenge, worker_cpu_cores, worker_memory):
         logger.exception(e)
         return e.response
 
+    if challenge.worker_image_url:
+        COMMON_SETTINGS_DICT["WORKER_IMAGE"] = challenge.worker_image_url
+
```

Comment:
This is wrong, you should not modify the `COMMON_SETTINGS_DICT`. This will change things for all challenges.

Can you test this after pushing evalai worker image on docker hub?

Reply:
You should change the task definition when it is being created!

Reply:
```task_def = task_definition.format(```